### PR TITLE
Add Barcode Scanner to loancomponent

### LINKED ISSUE #74: Create UX flow for adding multiple loans at a time.
## Description
There should be a way to assign multiple loans at the same time (e.g a whole class).

## Acceptance criteria:
- [ ] assign title
- [ ] scan barcode for book
- [ ] scan barcode for user
- [ ] Update wiki when feature is complete

## Context
See branch 74-multiple-loans for status update.

### PR DESCRIPTION
**Pull request by te4-adrian-almetun-smeds**
## ⚙️ What issue did you implement or fix?
Fixes #74

## 📜 Summary
- Added the ability to lend out a book using the barcodescanner component
- Improved UI flow for lending out books
- Fixed up some tests

## 📝 Type of change

Please delete options that are not relevant.

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 This change requires a documentation update

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Te4-Jacob-sandstrom — COMMENTED** [review]
rebaseing might fix deepscan

**Review by te4-alexander-kjellberg1 — DISMISSED** [review]
I see no problems with this.

**Comment by te4-adrian-almetun-smeds** [comment]
#508

File: frontend/cypress/integration/loanBooks.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-    it('Loans a book for a student on Admin Loans view', () => {
-        cy.get('[data-cy=student_barcode]').type('2819');
-        cy.get('[data-cy=book_barcode]').type('2819');
-        cy.get('[data-cy=loanOutBook]').click();
-        cy.get('[type=reset]').click();
-    })
-})
+  it('Loans a book for a student on Admin Loans view', () => {
+    cy.get('[data-cy=student_barcode]').type('2819');
+    cy.get('[data-cy=book_barcode]').type('2819');
```
Comment (te4-alexander-kjellberg1) [inline frontend/cypress/integration/loanBooks.js:10]: Nice, this was my favorit part! I liked that you changed from 4 to 2 spaces

**Review by te4-alexander-kjellberg1 — APPROVED** [review]

**Review by te4-dennis-christensen — COMMENTED** [review]
why is codeclimate failing?

**Review by Te4-Jacob-sandstrom — APPROVED** [review]

**Comment by te4-adrian-almetun-smeds** [comment]
> why is codeclimate failing?

Similar code.. 508 should fix it eventually.

**Pull request merged**